packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts

```diff
@@ -72,6 +75,10 @@ export abstract class BaseTemplateCommand extends SfdxCommandletExecutor<
     taskViewService.addCommandExecution(execution, cancellationTokenSource);
   }
 
+  protected runPostCommandTasks(outputDir: string) {
```

Comment:
I started to use baseTemplateCommand since I got everything I needed for Function creation. Now I see functions being more and more different - running post command task for example is only for functions. I've tried to add this empty method to base class to still use baseTemplateCommand. The other option is to not use baseTemplates and have execute and build implemented for functions separately.

Reply:
I think this is fine for now. When it's moved to templates lib, we could probably move this to yeoman generator https://yeoman.io/authoring/dependencies.html

Reply:
Why can't we accomplish this by using a `PostconditionChecker` ?

Reply:
OverwriteComponentPrompt and ConflicDetection should kick in before the command is executed, but in this case we want the task to executed after the command execution is fully done.